test/e2e/common/volumes.go

```diff
@@ -75,7 +75,7 @@ var _ = framework.KubeDescribe("GCP Volumes", func() {
 	////////////////////////////////////////////////////////////////////////
 
 	framework.KubeDescribe("NFSv4", func() {
-		It("should be mountable for NFSv4 [Volume]", func() {
+		It("should be mountable for NFSv4 [sig-storage]", func() {
```

Comment:
True, but that is a result of the implementation. Eg. `SIGxyzzyDescribe()` could be implemented in the _e2e_ directory or in _framework just to handle the case of some SIG-xyzzy tests residing in a different dir.